### ISSUE TITLE
Add Toerisme Vlaanderen as corporate member

### ISSUE THREAD

**Issue by joostschouppe** [osmbe/website#160]
In Dutch the alt text should be Toerisme Vlaanderen (https://toerismevlaanderen.be/), in English Visit Flanders (https://www.visitflanders.com)
![State_of_the_art_naked_version](https://user-images.githubusercontent.com/10122639/158201581-711a0c1b-aa21-4a86-a1b1-cebc62ad1316.png)
![Vlaanderen-is-toerisme_logo naakt](https://user-images.githubusercontent.com/10122639/158201585-2cc6148c-1990-42e8-9457-61bb755e5151.png)

French version should follow English one I guess

**Comment by jbelien** [comment]
Ready in #160 but there are 2 "issues":

- There is no logic for handling different information about a corporate member depending on the language ; so I used Toerisme Vlaanderen
- We need a lighter version of the logo to display on the dark background (see below)

![image](https://user-images.githubusercontent.com/1150563/158674883-db01765b-dd4d-4d40-a6ad-d56b9f313c6f.png)

**Comment by joostschouppe** [comment]
I discussed with TV. They prefer "always English" to "always Dutch". Here are a JPG and a PNG that should work on a dark background: https://user-images.githubusercontent.com/10122639/158782080-e6c68c93-c661-4fd5-a5de-6c4c9da2d6b2.jpg  https://user-images.githubusercontent.com/10122639/158782076-89417be1-2f11-4e4c-9576-3c0da631dc47.png
